Source/CarthageKit/BinaryProject.swift

```diff
@@ -2,8 +2,6 @@ import Foundation
 import Result
 import SPMUtility
 
-import struct Foundation.URL
```

Comment:
If per-type imports are preferred, we should add `Foundation.JSONDecoder` and `Foundation.Scanner`.

Reply:
I understand that `Foundation.URL` is required to disambiguate against `SPMUtility.URL`.